GUI/MetaDraw.xaml.cs

```diff
@@ -383,7 +383,7 @@ private async void loadFilesButton_Click(object sender, RoutedEventArgs e)
             prgsFeed.IsOpen = true;
             prgsText.Content = "Loading spectra file...";
 
-            var slowProcess = Task<MsDataFile>.Factory.StartNew(() => spectraFileManager.LoadFile(spectraFilePath, null, null, false, false, new CommonParameters()));
+            var slowProcess = Task<MsDataFile>.Factory.StartNew(() => spectraFileManager.LoadFile(spectraFilePath, new CommonParameters()));
```

Comment:
I think this changes the filtering. The original `LoadFile` method call used no peak filtering (`null, null, false, false`). Now it uses default `CommonParameters `settings, which include peak filtering (top 200 peaks, etc.)